aspnetcore/performance/response-compression/sample/Program.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace ResponseCompressionSample
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var host = new WebHostBuilder()

```

Comment:
Ah, but the reason its not in this sample is that we don't recommend that devs use this middleware when behind IIS. Also, it's not necessary to show how the middleware works, and there's nothing different about how the middleware works when the app is hosted behind IIS (correct?).